src/styles/variables.scss

```diff
@@ -25,6 +25,7 @@ $color-grey-light: #babac0;
 $color-grey-medium: #373737;
 $color-grey-dark: #787878;
 $color-grey-border: #5e5e5e;
+$color-grey-border-avatar: #5a5a5a;
```

Comment:
This color is in 99% similar to the above one. I guess this is a mistake on the UX level. I suggest to align with Tsvetan.